DP02_Galaxy_Cluster_Weak_Lensing.ipynb

```diff
@@ -0,0 +1,769 @@
+{
```

Comment:
<ul><li>If possible, add a reference to learn more about the red sequence method to identify cluster members. </li><li>"See also examples here": Add a more descriptive name/phrase to the hyperlink, not just the word "here". </li><li>(notebook 01): use Notebooks full name and link to rendered version. </li><li>Why is the zeropoint 31.4? </li><li>Explain more about the derivation of the error formula as approximately the inverse of the SNR. That'd help understand the multiplicative factor <code>fac</code> too.  </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/228/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
<ol><li>Added references.</li><li>Adjusted</li><li>Adjusted</li><li>The zeropoint is given in the notebook 01.</li><li>Adjusted</li></ol>